src/CommunityToolkit/Xamarin.CommunityToolkit/Views/MediaElement/MediaElement.shared.cs

```diff
@@ -124,6 +128,8 @@ public double Volume
 
 		internal event EventHandler PositionRequested;
 
+		internal event EventHandler VolumeRequested;
```

Comment:
Seems this event is not used
Should we remove it?

Reply:
It should be used on Android - the player doesn't expose volume changes programmatically so this ensures that if you query the Volume it updates the current value from the MediaPlayer. The event is not required on all platforms - for example on iOS there is an event which fires when the native control volume changes and this is updated in the MediaElement control's VolumeProperty. My implementation on Android has got lost in a merge somewhere so I'll add it back in.